### PR TITLE
no-gui_0.0.0.4.0.py

### PR DESCRIPTION
**Pull request by sergoum**

### CODE REVIEW

File: no-gui 0.0.0.3.2.py [hunk 1]
```diff
@@ -1,20 +1,71 @@
...
+        #here go the rest of the configs
+    reset()
     loaded = True
 print("done")
 print("type help for help")
 while True:
     inpt = input("@" + pcname + ":")
     inptlen = len(inpt.split())
     if inpt == 'help':
-        print("current commands: \nhelp: display all the commands and meanings\nbasevar: sets one of the basevars to a new value. Can use --pcname to change the pcname.")
```
Comment (rexillion0) [inline no-gui 0.0.0.3.2.py:14]: remove this

File: no-gui 0.0.0.3.2.py [hunk 2]
```diff
@@ -1,20 +1,71 @@
...
+    elif inpt == 'listapps':
+        print("clock")
     elif inptlen <= 3:
         if (inpt.split())[0] == 'basevar':
             if (inpt.split())[1] == '--pcname':
                 pcname = (inpt.split())[2]
+                #try:
+                #    pcname = (inpt.split())[2]
+                #except:
+                #    print("To change your pc's name, you have to specify another name.")
```
Comment (rexillion0) [inline no-gui 0.0.0.3.2.py:65]: remove these in comments

**Review by rexillion0 — CHANGES_REQUESTED** [review]

**Comment by rexillion0** [comment]
untill u fix

**Comment by sergoum** [comment]
what u mean?

File: no-gui 0.0.0.3.2.py [hunk 3]
```diff
@@ -1,20 +1,71 @@
...
+                #    print("Do you need further help with this command?(y/n)")
+                #    temp = input()
+                #    if temp == "y":
+                #        print("This command is used like this: basevar --pcname putyournamehere(no spaces are allowed)")
     else:
-        print("error code 1 - unknown command - invalid syntax")
```
Comment (sergoum) [inline no-gui 0.0.0.3.2.py:20]: i removed becuase i didnt really like that

File: no-gui 0.0.0.3.2.py (hunk 1 above)
Comment (sergoum) [inline no-gui 0.0.0.3.2.py:14]: i made a var named help instead(line 10)

File: no-gui 0.0.0.3.2.py [hunk 4]
```diff
@@ -1,20 +1,71 @@
 import time as ti
-print("Welcome to Voluro.py no-gui 0.0.0.3.2")
+import keyboard
+print("Welcome to Voluro.py no-gui 0.0.0.4.0")
 loaded = False
 print("loading")
 while loaded != True:
-    pcname = "Voluro"
```
Comment (sergoum) [inline no-gui 0.0.0.3.2.py:6]: i moved this down a bit

File: no-gui 0.0.0.3.2.py [hunk 5]
```diff
@@ -1,20 +1,71 @@
 import time as ti
-print("Welcome to Voluro.py no-gui 0.0.0.3.2")
```
Comment (sergoum) [inline no-gui 0.0.0.3.2.py:2]: i changed, because this version is updated

**Review by sergoum — COMMENTED** [review]
this is an older version

File: no-gui 0.0.0.3.2.py (hunk 5 above)
Comment (rexillion) [reply no-gui 0.0.0.3.2.py:2]: ok

File: no-gui 0.0.0.3.2.py (hunk 4 above)
Comment (rexillion) [reply no-gui 0.0.0.3.2.py:6]: good

File: no-gui 0.0.0.3.2.py (hunk 1 above)
Comment (rexillion) [reply no-gui 0.0.0.3.2.py:14]: ok it is easier but abit more heavy

File: no-gui 0.0.0.3.2.py (hunk 3 above)
Comment (rexillion) [reply no-gui 0.0.0.3.2.py:20]: nonnnonono

**Review by rexillion — CHANGES_REQUESTED** [review]

**Comment by sergoum** [comment]
so is ok?

File: no-gui 0.0.0.3.2.py (hunk 1 above)
Comment (sergoum) [reply no-gui 0.0.0.3.2.py:14]: ok we can keep it the way you made it

File: no-gui 0.0.0.3.2.py (hunk 3 above)
Comment (sergoum) [reply no-gui 0.0.0.3.2.py:20]: ok fine

**Comment by rexillion** [comment]
It will be merged for upgrade purposes

**Pull request closed without merging**